src/librustdoc/fold.rs

```diff
@@ -1,7 +1,12 @@
 use crate::clean::*;
+use regex::Regex;
 
 pub struct StripItem(pub Item);
 
+lazy_static! {
```

Comment:
Please use [lazycell](https://docs.rs/lazycell/1.2.1/lazycell/) instead.

Reply:
I'll do it, but I'm just curious, why? I like to know :)